src/components/Portfolio/experience.ts

```diff
@@ -1,6 +1,20 @@
 const { uuid } = require('uuidv4');
 
-const frontEnd = [
+type UUID = {
+  key: string;
+}
+
+type Knowledge = {
+  title: string;
+  icon: string;
+} & Pick<UUID, 'key'>
+
+type Experience = {
+  date: string;
+  description: string
```

Comment:
To be more precise the `description` field should be renamed to reflect what it actually is, a i18n-key. Perhaps rename field key to `translationKey`?